### PR TITLE
Add image for kube-dns version 1.22.5

### PR DESCRIPTION
**Pull request by kl52752**

### CODE REVIEW

File: k8s.gcr.io/images/k8s-staging-dns/images.yaml [hunk 1]
```diff
@@ -271,6 +281,7 @@
     "sha256:a507030925ea65b49e829611bc05c99b21d8333d690fc46e017ca1ab1bdc5dc6": ["1.22.0"]
     "sha256:1ab4f964a2ab7a06e220fbd0eea696ad60eec4d35552be459b030c48675ed656": ["1.22.1"]
     "sha256:ef44939a47d3a9451ba5b946496e0a40eb7f9c73a6e573a55b5a6dbfc577d97e": ["1.22.2"]
+    "sha256:47a7e7911397b220febcafdf8fbcb58619432de71d20f799ec9617e42f51c503": ["1.22.5"]
```
Comment (dpasiukevich) [inline k8s.gcr.io/images/k8s-staging-dns/images.yaml:284]: could you also add hashed for sidecar container as well?

Comment (kl52752) [reply k8s.gcr.io/images/k8s-staging-dns/images.yaml:284]: yes, it is added in this PR

File: k8s.gcr.io/images/k8s-staging-dns/images.yaml [hunk 2]
```diff
@@ -376,3 +388,6 @@
     "sha256:ddb7ee7b6ded62ccfd9cbeaafb87c9eb8b440a0514d0545709c5dd1b5bffbd4f": ["1.22.0"]
     "sha256:5bebd19c21c82a9f19780321f9e6f2b6029299e0f656dc64f5c918d960edca7c": ["1.22.1"]
     "sha256:7b35fdaa9fac8e7e638f6040cd0d49ffe9804dd3f12a0bf124664f1df18e95cf": ["1.22.2"]
+    "sha256:8a2ff9817467f9f2d4559fe5f28664264cf30d8ce71b0da0886e81774c41dd96": ["1.22.3"]
```
Comment (MrHohn) [inline k8s.gcr.io/images/k8s-staging-dns/images.yaml:391]: Humm, this and the following hashes look the same as the ones from `k8s-dns-node-cache` - could you double check?

**Review by MrHohn — CHANGES_REQUESTED** [review]

**Comment by kl52752** [comment]
The PR is duplicated

**Pull request closed without merging**